pkg/controller/kubelet-config/helpers.go

```diff
@@ -155,7 +155,7 @@ func updateOriginalKubeConfigwithNodeConfig(node *osev1.Node, originalKubeletCon
 	case osev1.DefaultUpdateDefaultReaction:
 		originalKubeletConfig.NodeStatusUpdateFrequency = metav1.Duration{Duration: osev1.DefaultNodeStatusUpdateFrequency}
 	case emptyInput:
-		return nil
+		originalKubeletConfig.NodeStatusUpdateFrequency = metav1.Duration{Duration: osev1.DefaultNodeStatusUpdateFrequency}
```

Comment:
nit: This case can be removed by appending the `emptyInput` to the above case as suggested

Reply:
I think it might be best to leave the case for empty just in case a user somehow messes with their config?